common/utils_unittest.cpp

```diff
@@ -126,9 +126,9 @@ UTILS_TEST(guessResolutionOverflow) {
             << "x"
             << std::numeric_limits<long>::max();
 
-    EXPECT_FALSE(guessResolution(sstream.str().c_str(), w, h));
-    EXPECT_EQ(w, 0);
-    EXPECT_EQ(h, 0);
+    EXPECT_TRUE(guessResolution(sstream.str().c_str(), w, h));
```

Comment:
hi @xuguangxin , do you mean that we should assign 0 to width, when it is greater than 3840, so as height ?